app/assets/stylesheets/darkswarm/shop_search.css.scss

```diff
@@ -0,0 +1,45 @@
+@import "mixins";
+@import "branding";
+@import "variables";
+
+.shop-searchbar {
+  background-color: $grey-100;
+  height: 5em;
+  padding: 1em 0;
+  margin-bottom: 1em;
+  position: relative;
+  z-index: 5;
+
+  input#search {
+    height: 3em;
+    border-radius: $radius-small;
+    border: solid 1px $grey-300;
+    margin: 0;
+    padding: 0 1em 0 2.75em;
+    width: 100%;
+    min-width: 0;
+    background: $white url("/assets/icn-search-grey.png") 1em center no-repeat;
+    font-size: 1em;
```

Comment:
same here, is this not a generic input box?

Reply:
Currently I'd say it's a very specific set of styles that will only be used on this input in this context...